### PR TITLE
TaskAssignments panel MVP refactoring & bugfix test migration

### PR DESCRIPTION
**Pull request by jhrcek**
Migrating several selenium bugfix tests we have to presenter gwt unit tests.
To enable this, TaskAssignments panel had to be refactored to MVP.

### CODE REVIEW

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskassignments/TaskAssignmentsPresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+        verify(viewMock).setDelegateButtonActive(false);
+        verify(viewMock, times(2)).enableDelegateButton(false);
+    }
+
+    @Test
+    public void emptyDelegationUserOrGroup_notAccepted() {
+        presenter.delegateTask("");
+
+        verify(viewMock).setHelpText("DelegationUserInputRequired");
```
Comment (Salaboy) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskassignments/TaskAssignmentsPresenterTest.java]: @jhrcek  Shouldn't be this -> Constants.INSTANCE.DelegationUserInputRequired() ?

**Comment by Salaboy** [comment]
+1 @jhrcek thanks a lot Jan!

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskassignments/TaskAssignmentsPresenterTest.java (hunk 1 above)
Comment (jhrcek) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskassignments/TaskAssignmentsPresenterTest.java]: @Salaboy You're right. I was thinking how to deal with the fact that Constants field in the TaskAssignmentsViewImpl is not initialized. I'll update the PR

**Comment by cristianonicolai** [comment]
+1, thanks @jhrcek

**Comment by csadilek** [comment]
Thanks.

master: fdce6b1328cedcd3e41eb44ff6c9b901a30c9a6b

**Pull request closed without merging**